File: Age distribution.cpp
#include <iostream>
#include <fstream>

using namespace std;

int main(int argc,char *argv[])
{
    fstream file (argv[1],ios::in);
    string buffer;
    
    while(!file.eof())
    {
        getline(file,buffer);
        if (atoi(buffer.c_str())>=0 && atoi(buffer.c_str())<=2)
        {
            cout<<"Still in Mama's arms"<<endl;
        }
        else if (atoi(buffer.c_str())>=3 && atoi(buffer.c_str())<=4)
        {
            cout<<"Preschool Maniac"<<endl;
        }
        else if (atoi(buffer.c_str())>=5 && atoi(buffer.c_str())<=11)
        {
            cout<<"Elementary school"<<endl;
        }
        else if (atoi(buffer.c_str())>=12 && atoi(buffer.c_str())<=14)
        {
            cout<<"Middle school"<<endl;
        }
        else if (atoi(buffer.c_str())>=15 && atoi(buffer.c_str())<=18)
        {
            cout<<"High school"<<endl;
        }
        else if (atoi(buffer.c_str())>=19 && atoi(buffer.c_str())<=22)
        {
            cout<<"College"<<endl;
        }
        else if (atoi(buffer.c_str())>=23 && atoi(buffer.c_str())<=65)
        {
            cout<<"Working for the man"<<endl;
        }
        else if (atoi(buffer.c_str())>=66 && atoi(buffer.c_str())<=100)
        {
            cout<<"The Golden Years"<<endl;
        }
        else {cout<<"This program is for humans"<<endl;}

    }
    file.close();
    return 0;
}

File: easy-level/Age distribution.py
import sys

def answer(age):
	if age in range(0,3):
		return('Still in Mama\'s arms')
	elif age in range(3,5):
		return('Preschool Maniac')
	elif age in range(5,12):
		return('Elementary school')
	elif age in range(12,15):
		return('Middle school')
	elif age in range(15,19):
		return('High school')
	elif age in range(19,23):
		return('College')
	elif age in range(23,66):
		return('Working for the man')
	elif age in range(66,101):
		return('The Golden Years')
	else:
		return('This program is for humans')


file= open("textinput.txt",'r')
for line in file:
    print(answer(int(line.rstrip())))


file.close()



